VisualizationBase/src/Scene.cpp

```diff
@@ -606,6 +606,7 @@ void Scene::removeAllOnZoomHandlers()
 int Scene::addOnZoomHandler(OnZoomHandler onZoomHandler, OnZoomHandlerRemove onZoomHandlerRemove)
 {
 	onZoomHandlers_.insert(addOnZoomHandlerId_, {onZoomHandler, onZoomHandlerRemove});
+	onZoomHandler(mainViewScalingFactor_);
```

Comment:
Hmm, why is this change necessary? Wouldn't it perhaps make more sense to call the handling code from the outside in whatever case you need this? Logically it doesn't make sense to me to call this here, but maybe you could convince me otherwise.